app/src/main/java/ch/epfl/sdp/appart/ad/AdViewModel.java

```diff
@@ -31,6 +34,7 @@
     private final MutableLiveData<String> adAdvertiserName = new MutableLiveData<>();
     private final MutableLiveData<String> adAdvertiserId = new MutableLiveData<>();
     private final MutableLiveData<List<String>> adPhotosRefs = new MutableLiveData<>();
+    private final MutableLiveData<List<String>> panoramasReferences = new MutableLiveData<>();
```

Comment:
maybe like panoramaReferences you can change the name of adPhotosRefs in adPhotosReferences or vice-versa.

Reply:
Yes !